### ISSUE TITLE
Cutting 1.2 Release

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#668]
It's probably time to cut a 1.2 release of DeepChem. Are there any last PRs people want merged into 1.2? Will wait a couple days to give people time to respond before cutting.

**Comment by lilleswing** [comment]
Cut away!

**Comment by rbharath** [comment]
Released! https://github.com/deepchem/deepchem/releases

@lilleswing Could you upgrade the docker image?

**Comment by rbharath** [comment]
Made a PR to update the conda version of DeepChem

https://github.com/omnia-md/conda-recipes/pull/802

Once that PR is merged in, we should be good to close this issue

**Comment by rbharath** [comment]
The conda PR for the 1.3 conda package has run into weird bugs that I'm not sure how to fix.

Would someone else might taking a look?

CC @peastman @lilleswing

**Comment by peastman** [comment]
It looks like a glibc compatibility issue:

```
ImportError: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /opt/conda/conda-bld/deepchem_1500434688872/_t_env/lib/python2.7/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)
```

As Levi said, travis recently moved from Ubuntu 12.04 to 14.04, so that's probably what caused the problem.  But I don't know why.  Issues like this usually work the other way: it works on newer OS versions, but not older ones.  In fact, searching for the error message turned up https://github.com/tensorflow/tensorflow/issues/53, which basically says exactly that: it should work on Ubuntu 14.04, but not earlier versions.

**Comment by lilleswing** [comment]
https://github.com/omnia-md/conda-recipes/pull/802

**Comment by rbharath** [comment]
Closing since we're about to cut 1.3
